### ISSUE TITLE
Add reviewers to registry data

### ISSUE THREAD

**Issue by jeroen** [ropensci/roregistry#27]
We would like to display on the user profile pages which packages a user has reviewed. What would be the best way to save the reviewers in the registry data?

**Comment by sckott** [comment]
Carl says codemeta will only get reveiwers if they are included in the DESCRIPTION file. And that is optional for authors. So we will need to add reviewers from the airtable list of reviewers (table "Reviews" and "Reviewers" [need github handles from here])

**Comment by maelle** [comment]
What info about reviewers do you want the registry to contain, only their GitHub handles?

**Comment by jeroen** [comment]
I think their ropensci handle from https://github.com/ropensci/roweb2/tree/master/content/authors which then contains their github handle.

**Comment by maelle** [comment]
Good idea. I'll need to make sure we have the right name in the Airtable for the folks that already have an rOpenSci handle (for other folks, we can assume the name is right).

I can see it'll mean listing on the website at some point. When we do that we'll need to make sure it's fine to add profile pages who might not be aware.

**Comment by maelle** [comment]
PR open at https://github.com/ropenscilabs/makeregistry/pull/10

At the moment this only gives the name and GitHub usernames as stored in Airtable, I need to look into slugified names.

**Comment by maelle** [comment]
@jeroen if reviewer GitHub names were added to the registry, would it be a good feature to add to R-universe package pages?
